contracts/cw-core/Cargo.toml

```diff
@@ -29,13 +29,13 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-std = { version = "1.0.0", features = ["ibc3"] }
-cosmwasm-storage = { version = "1.0.0" }
-cw-storage-plus = "0.13"
-cw2 = "0.13"
-cw-utils = "0.13"
-cw20 = "0.13"
-cw721 = "0.13"
+cosmwasm-std = { version = "1.1.0", features = ["ibc3"] }
+cosmwasm-storage = "1.1.0"
```

Comment:
Minor note, we can use "1.1" and "0.15" rather than specify patch I believe.

Reply:
I opted to be explicit, since we don't follow semver over correctly anyway... But sure, I'll remove the patch versions.